docs/user-guide/introduction/shortcuts.njk

```diff
@@ -307,6 +307,11 @@ title: Shortcuts
       <td style="text-align: center;"><kbd>Shift</kbd><kbd>↑</kbd></td>
       <td style="text-align: center;"><kbd>⇧</kbd><kbd>↑</kbd></td>
     </tr>
+    <tr>
+      <td>Rename selected layer</td>
+      <td style="text-align: center;"><kbd>Alt</kbd><kbd>N</kbd></td>
+      <td style="text-align: center;"><kbd>⌘</kbd><kbd>N</kbd></td>
```

Comment:
I tried on Mac and `⌘N` opens a new browser window (as usual). `Alt + N` works to rename the selected layer.

Reply:
Thanks @belen-albeza! You're right, I've corrected it! Can you check now?